assets/src/containers/App.js

```diff
@@ -5,11 +5,27 @@ import Course from './Course'
 import WarningBanner from '../components/WarningBanner'
 import AlertBanner from '../components/AlertBanner'
 import { Helmet } from 'react-helmet'
-import useGoogleAnalytics from '@tl-its-umich-edu/react-ga-onetrust-consent'
+import {useGoogleAnalytics, useUmConsent} from '@tl-its-umich-edu/react-ga-onetrust-consent'
 
 function App (props) {
-  const { user, gaId, cspNonce, oneTrustScriptDomain } = props
-  useGoogleAnalytics({ googleAnalyticsId: gaId, nonce: cspNonce, oneTrustScriptDomain })
+  const { user, gaId, cspNonce } = props
+  const { gaInitialized, gaHandlers } = useGoogleAnalytics({ googleAnalyticsId: gaId, nonce: cspNonce })
+  const { umConsentInitialize, umConsentInitialized } = useUmConsent()
+
+  if (
+    !umConsentInitialized &&
+    gaInitialized &&
+    gaHandlers.onConsentApprove &&
+    gaHandlers.onConsentReject
+  ) {
+    const consentParams = {
+      developmentMode: false,
+      alwaysShow: false,
+      onConsentApprove: gaHandlers.onConsentApprove,
+      onConsentReject: gaHandlers.onConsentReject,
+    };
+    umConsentInitialize(consentParams)
+  }
```

Comment:
I see the with Loophole or ngrok the cookie doesn't set, looks like further diagnosis on this could be local thing and not non-prod/prod issue.

Reply:
I am fine merging it now

Reply:
If I remember correctly with oneTrust approach last time we never tested the PR from Localhost, the PR changes were set on Openshift Non-prod and we were just testing from there. So I don't know exactly why we did that last time, so to me we can merge and test it from Openshift